### PR TITLE
add basic routing

### PR DESCRIPTION
**Pull request by bmorelli25**
Adds and enables a "Create room" and "Join room" button. Logic is basic and only minimal validation exists.

### CODE REVIEW

File: pages/index.tsx [hunk 1]
```diff
@@ -1,25 +1,65 @@
...
+
+  useEffect(() => {
+    if(!newRoom) {
+      setNewRoom(roomGen());
+    }
+  }, [newRoom]);
+
+  const roomGen = () => {
+    function a() {
+      return String.fromCharCode(65+Math.floor(Math.random() * 26));
```
Comment (dstrizzi) [inline pages/index.tsx:19]: Nice! This is neat

File: pages/index.tsx [hunk 2]
```diff
@@ -1,25 +1,65 @@
...
+    }
+    return a().concat(a(),a(),a());
+  }
+
+  const onChange = (e) => {
+      setJoinRoom(e.target.value.replace(/[1-9]/g, "").toUpperCase())
```
Comment (dstrizzi) [inline pages/index.tsx:25]: Obviously fine for now, but should add a ticket or something to update this to letters only.

**Review by dstrizzi — APPROVED** [review]
LET'S GOOOOOOO.

Tested locally and approved.

File: pages/index.tsx (hunk 2 above)
Comment (bmorelli25) [reply pages/index.tsx:25]: does it not already? I can only type letters.

File: pages/index.tsx (hunk 1 above)
Comment (bmorelli25) [reply pages/index.tsx:19]: Yeah. There's probably a way to clean up the Concat part of this, but we can worry about that later.

**Comment by bmorelli25** [comment]
I don't have merg perms so feel free to send it 🚢

**Comment by dstrizzi** [comment]
Symbols need to be removed as well

File: pages/index.tsx (hunk 1 above)
Comment (bmorelli25) [reply pages/index.tsx:19]: ++ to symbols. I can type those and letters

File: pages/index.tsx (hunk 2 above)
Comment (bmorelli25) [reply pages/index.tsx:25]: ++ to symbols. I can type those and letters

**Pull request merged**